### PR TITLE
feat(fig/fig@latest): update spec

### PR DESCRIPTION
**Pull request by mschrage**
PR generated automatically from push-to-fig-autocomplete-action.

### CODE REVIEW

File: src/fig/fig@latest.ts [hunk 1]
```diff
@@ -0,0 +1,1395 @@
...
+  debug: (debug) =>
+    edit(debug.subcommands, {
+      logs: (logs) =>
+        edit(logs.args, {
+          files: {
+            isVariadic: true,
+          },
+        }),
+    }),
+});
```
Comment (fedeci) [inline src/fig/fig@latest.ts:1395]: /cc @SeparateRecords this can be removed, right?

Comment (SeparateRecords) [reply src/fig/fig@latest.ts:1395]: Looks like the edits need to stay for now - the `internal` subcommand is aliased to `_`, and the `fig debug logs [files...]` arg is variadic, but it doesn't look like that's in the spec. I'd imagine that's easy enough to fix on the rust side though!

Comment (fedeci) [reply src/fig/fig@latest.ts:1395]: The variadic prop is added when `Arg::multiple_values(true)`, aliases are still unhandled.
/cc @grant0417

Comment (fedeci) [reply src/fig/fig@latest.ts:1395]: Opened a PR for subcommand aliases.

**Pull request closed without merging**